src/biokbase/narrative/tests/test_jobcomm.py

```diff
@@ -393,89 +453,145 @@ def test_start_job_status_loop__cell_ids(self):
     # Lookup all job states
     # ---------------------
 
-    def check_lookup_job_states(self, output_states=None, job_id_list=[]):
+    @mock.patch(CLIENTS, get_mock_client)
+    def check_job_output_states(
```

Comment:
numerous requests return the job output states, so I made this useful helper to do all the output testing